docs/Makefile

```
@@ -6,18 +6,37 @@ SHELL := /bin/bash
+SPHINXBUILD   ?= .venv/bin/sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
+VENV          = .venv
+PYTHON        = python3
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+.PHONY: help Makefile install clean-venv
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)